### ISSUE TITLE
In app lockdown mode, port bypass rules are not honoured

### ISSUE THREAD

**Issue by ignoramous** [celzero/rethink-app#721]
A user says,

> Hey, I have put telegram in lockdown mode with port 443 bypassed but it's still being blocked

Ref: https://t.me/rethinkdns/61790

**Comment by ignoramous** [comment]
Doesn't happen always, but intermittently... Which is more worrying:

<details>
<summary>wildcard bypass rule `*.*.*.*:443` leak with Telegram</summary>
It's occured again just now see the time. The lockdown rule has been in effect since the 16th, I don't use this version of telegram app often as I have multiple telegrams apps, I just opened it today and it refused to connect, checked the log and all the connection attempts were block, I had to "Bypass App Rule" and add the specific entry and it establishes connection. See the red and then after "Bypass App Rule" added its grey, establishing connection a few seconds later, and I still have wildcard bypass rule *.*.*.*:443 for that specific app.

From: https://t.me/rethinkdns/61810

![IMG_20221222_004213_774](https://user-images.githubusercontent.com/852289/208984841-822eaff6-6eeb-49e6-83e1-fbca7056592d.jpg)
</details>
